index.js

```diff
@@ -0,0 +1,156 @@
+});
+
+let rtmConnectionOpened = false;
+
+rtm.on(RTM_EVENTS.MESSAGE, function handleRtmMessage(message) {
+
+    if (!message.text) {
+        return;
+    }
+
+    // forgive trailing spaces and ignore case
+    const normalizedMessage = message.text.trim().toLowerCase();
+    let now = false;
+    let tomorrow = false;
+
+    // only respond to messages where we are mentioned
+    if (normalizedMessage.indexOf(`<@${botID}>`) === -1) {
+        return;
+    }
+
+    // private channel ID isn't available on start, so respond on the channel
+    // we're mentioned on.
+    botChannel = message.channel;
+
+    if (normalizedMessage.indexOf('weather now') !== -1) {
+        now = true;
+    }
+
+    if (normalizedMessage.indexOf('weather tomorrow') !== -1) {
+        tomorrow = true;
+    }
+
+    if (!now && !tomorrow) {
+        if (rtmConnectionOpened) {
+            if (normalizedMessage.indexOf('help') !== -1) {
+                rtm.sendMessage(`<@${message.user}>, try @surry-interview 'weather now' or 'weather tomorrow'`, botChannel);
+            } else {
+                rtm.sendMessage(`<@${message.user}>, I did not understand that command, try @surry-interview 'weather now' or 'weather tomorrow'`, botChannel);
+                console.log('Ignoring unknown command: ' + normalizedMessage.substring(0, 20) + '...');
+            }
+        }
+        return;
+    }
+
+    if (now) {
+        util.getCurrentWeatherReport().then((json) => {
+            if (rtmConnectionOpened) {
+                if (json.currently) {
+                    let message = util.generateCurrentSummary(json.currently);
+                    rtm.sendMessage(message, botChannel);
+
+                    const now = new Date().getTime();
+                    // This url can be used to grab a frame from a DDOT (DC) traffic cam!
+                    // This can be used to give a rough visual of the weather. The camera is
+                    // at the intersection of 14th St and Independence St NW. Looks like the current
+                    // time in milliseconds since 1970 needs to be passed in to get the latest images.
+                    // The RTM client doesn't allow sending attachments (see https://api.slack.com/rtm),
+                    // but the Slack URL unfurling functionality automatically includes the traffic cam frame!
+                    // See http://app.ddot.dc.gov/ for more info.
+                    const trafficCamUrl = `http://ie.trafficland.com/v1.0/200146/full?system=ddot&pubtoken=b4c0b819f66741b99fdb17c963f47e3bafd09b494019ee9624f1d38641ebc6bf&refreshRate=2000&t=${now}`
+
+                    // the RTM client unfortunately doesn't support formatting of URLs with <URL|human-friendly-string>
+                    rtm.sendMessage(`Here's a snapshot of the current weather at 14th St and Independence St in DC: ${trafficCamUrl}`, botChannel);
+                }
+            }
+        }).catch((error) => {
```

Comment:
nit: i don't think we need an anonymous function here if we're just going to pass the args through
```
.catch(console.error)
```